README.md

```diff
@@ -97,27 +97,43 @@ The options that we support for calculation are as follows:
+  --report-type <report-type>                 type of report, "individual", "summary", "subject-list"
   -m, --measure-bundle <measure-bundle>       path to measure bundle
-  -p, --patient-bundles <patient-bundles...>  paths to patient bundles. Required unless output type is dataRequirements
-  -f, --as-patient-source                     Load bundles by creating cql-exec-fhir PatientSource to pass into library calls
+  -p, --patient-bundles <patient-bundles...>  paths to patient bundles. Required unless output type is dataRequirements. Note: cannot be used with --patient-ids.
+  --patient-ids <ids...>                      (with --fhir-server-url) A list of patient ids an AsyncPatientSource will use to query a fhir server for patient data. Note: cannot be used with --patient-bundles.
+  --as-patient-source                         Load bundles by creating cql-exec-fhir PatientSource to pass into library calls
+  --fhir-server-url <url>                     (with --as-patient-source) Loads bundles into an AsyncPatientSource which queries the passed in fhir server url for patient data
   -s, --measurement-period-start <date>       start date for the measurement period, in YYYY-MM-DD format (defaults to the start date defined in the Measure, or 2019-01-01 if not set
                                               there)
   -e, --measurement-period-end <date>         end date for the measurement period, in YYYY-MM-DD format (defaults to the end date defined in the Measure, or 2019-12-31 if not set there)
-  -a, --vs-api-key <key>                      API key, to authenticate against the valueset service to be used for resolving missing valuesets
-  -c, --cache-valuesets                       Whether or not to cache ValueSets retrieved from the ValueSet service (default: false)
+  --vs-api-key <key>                          API key, to authenticate against the valueset service to be used for resolving missing valuesets
+  --cache-valuesets                           Whether or not to cache ValueSets retrieved from the ValueSet service (default: false)
   -h, --help                                  display help for command
 ```
 
 E.g.
 
 ```bash
-fqm-execution -o reports -m /path/to/measure/bundle.json -p /path/to/patient1/bundle.json > reports.json
+Generate a MeasureReport by calculating a measure on a patient bundle:
+  - fqm-execution reports -m /path/to/measure/bundle.json -p /path/to/patient/bundle.json > reports.json
+
+Generate a MeasureReport by calculating a measure on multiple patient bundles:
+  - fqm-execution reports -m /path/to/measure/bundle.json -p /path/to/patient/bundle1.json /path/to/patient/bundle2.json > reports.json
+
+Generate a MeasureReport by calculating a measure on a patient source which pulls its patient data from a fhir server:
+  - fqm-execution reports -m /path/to/measure/bundle.json --as-patient-source --fhir-server-url http://example.com --patient-ids test_id_1 test_id_2 > reports.json
```

Comment:
Where do you test the fhir-server?  Have you tried using https://cqm-sandbox.alphora.com/cqf-ruler-r4/fhir/ ?

Reply:
@p9g Currently we are testing with our own fhir server [deqm-test-server](https://github.com/projecttacoma/deqm-test-server). Happy to give it a shot with this server as well.

Reply:
We gave this a test with cqf-ruler, and it pulled the data as expected :)